properties/basic.py

```diff
@@ -392,18 +392,12 @@ def validate(self, instance, value):
 
     @staticmethod
     def to_json(value):
-        if value is np.nan:
-            return 'nan'
-        if value is np.inf:
-            return 'inf'
+        if np.isnan(value) or np.isinf(value):
+            return str(value)
         return value
 
     @staticmethod
     def from_json(value):
-        if value == 'nan':
-            return np.nan
-        if value == 'inf':
-            return np.inf
         return float(str(value))
```

Comment:
This is something that is hung over from an earlier version where a string object could potentially make it this far. We were using it in that backend to validate web-request handling.

`http://website.com/page?number=1.3` --> `float(str(number))` etc.

Thanks for cleaning.